Gemfile

```
@@ -1,4 +1,3 @@
 source 'https://rubygems.org'
 
 gem 'cocoapods'
-gem 'fastlane'
```

The tests are failing because we still need `xcpretty`

```suggestion
gem 'xcpretty'
```

---

d’oh. Fixed.